modules/c++/math.poly/include/math/poly/Fit.h

```diff
@@ -6,6 +6,7 @@
 #include <math/linear/Matrix2D.h>
 #include <math/linear/VectorN.h>
 #include <numeric>
+#include <sstream>
```

Comment:
Even though you are almost certainly going to get these includes via other headers, should include sys/Conf.h and except/Exception.h since you're using them here.